Lib/test/datetimetester.py

```diff
@@ -6018,6 +6018,17 @@ class TZInfoSubclass(tzinfo):
                 with self.subTest(arg=arg, exact=exact):
                     self.assertFalse(is_tzinfo(arg, exact))
 
+    def test_date_from_date(self):
+        year = 1993
+        month = 8
+        day = 26
+
+        for macro in [0, 1]:
+            with self.subTest(macro=macro):
+                d = _testcapi.get_date_fromdate(year, month, day, macro)
+
+                self.assertEqual(d, date(1993, 8, 26))
```

Comment:
and because I don't like the duplicated code, you could use that.

```python
self.assertEqual(d, date(year, month, day))
```
I think you could also compute one time the expected date. and because I like to be explicit...

```python
def test_date_from_date(self):
    expected_date = date(1993, 8, 26)
    for macro in (0, 1):
        with self.subTest(macro=macro):
            new_date = _testcapi.get_date_fromdate(expected_date.year, expected_date.month, expected_date.day, macro)
            self.assertEqual(new_date, expected_date)
```

@pganssle ?